### ISSUE TITLE
Remove jsnext:main

### ISSUE THREAD

**Issue by magicmark** [cspotcode/outdent#1]
`jsnext:main` is deprecated:
https://github.com/rollup/rollup/wiki/jsnext:main

as a result, it is also causing issues when I try to consume this package with the airbnb default settings:

```
error: Missing file extension "es" for "outdent" (import/extensions) at src/create-bundle.js:12:21:
  10 | import glob from 'glob';
  11 | import mkdirp from 'mkdirp';
> 12 | import outdent from 'outdent';
     |                     ^
  13 | import camelcase from 'camelcase';
  14 | import path from 'path';
  15 |
```

**Comment by cspotcode** [comment]
Thanks for the PR and for reporting the issue.

The wiki page you linked doesn't exist, but I found this page: https://github.com/rollup/rollup/wiki/pkg.module

It looks like `jsnext:main` / `module` should point to a version of the code with ES6 import and export statements, but otherwise the code should be transpiled the same way.  Is that correct?

I updated `jsnext:main` and added `module`; they both point to a compiled version of the library that's been downleveled to ES5 but uses ES6 exports.  The file extension is `.js`, not `.es`.  (here's the commit: 7c9a039)

I published outdent@0.4.0-rc2.  Can you install it and let me know if it fixes your problem?  (You'll have to manually install "outdent@rc" because it's a pre-release version)

**Comment by jrolfs** [comment]
@cspotcode this is fixed for me with `0.4.0-rc2`.

To clarify, this issue reported is actually with ESLint + `eslint-plugin-import` as configured in Airbnb's ESLint configuration. I was using the same configuration and this resolves the extension issue.

Thanks!

**Comment by cspotcode** [comment]
That's great to hear, I'll publish `0.4.0` shortly!
